### ISSUE TITLE
[KR specific] Prevent donation from same country

### ISSUE THREAD

**Issue by revi** [liberapay/liberapay.com#1280]
I know East Asia is a rare area for Liberapay but prevention is better than the hotfix, right?

South Korea has a legislation for electronic money transmission and Paypal just restricted the transaction for the account with the same country:

* Sending to Paypal accounts with the country "S. Korea": Fails
* Sending to the rest of the world: Success

## Help docs [link, might be different for you so screenshot attached](https://www.paypal.com/kr/smarthelp/article/how-do-i-send-a-payment-through-paypal-faq1684)
<img width="733" alt="image" src="https://user-images.githubusercontent.com/7630875/46919990-809c6a00-d022-11e8-9645-0b0179b8123a.png">

I've heard Japan was the case here but couldn't confirm, so did not include in [* specific].

**Comment by Changaco** [comment]
I don't think we have enough users in South Korea to warrant adding a warning on our side, we should just make sure that we properly handle the errors that PayPal returns (#1277).

**Comment by soohwa** [comment]
@revi Hello, maybe you can understand this pages:
- https://blog.payoneer.com/ko/how_to-ko/%eb%8c%80%ea%b8%88-%eb%b0%9b%eb%8a%94-%eb%b0%a9%eb%b2%95/how-to-get-paid-by-companies-worldwide-with-payoneers-global-payment-service/
- https://blog.naver.com/payoneer_kr

**Comment by revi** [comment]
@soohwa I can read and write Korean, but I am not sure what you are asking for. (Sorry for the late reply!)

**Comment by soohwa** [comment]
@revi thx for your answer ^^ I was just wondering if payoneer could become one of the payment processors of liberapay and if payoneer was operating in South Korea, i.e. seeing funds from or to South Korea, and possibly from South Korea to South Korea.

**Comment by SettingDust** [comment]
> In fact I think Payoneer is not suitable for liberapay: [paynow.payoneer.com/pagecontent.payoneer?rsckey=ProcessorTypeDebitCard.Terms&pid=-1#_Toc532218206](https://paynow.payoneer.com/pagecontent.payoneer?rsckey=ProcessorTypeDebitCard.Terms&pid=-1#_Toc532218206)
> 
> Sorry about the noise.

What's the content before? Seems there isn't donation related now?
